SingularityS3Base/src/main/java/com/hubspot/singularity/s3/base/config/SingularityS3Configuration.java

```diff
@@ -131,7 +131,7 @@ public void setLocalDownloadPath(String localDownloadPath) {
     this.localDownloadPath = localDownloadPath;
   }
 
-  public long getS3ChunkDownloadTimeoutMillis() {
+  public Long getS3ChunkDownloadTimeoutMillis() {
```

Comment:
I'd say let's change this to `int`. The max value of a Java integer in milliseconds is a little more than 24 days, which should be more then enough for this timeout.